README.md

```diff
@@ -184,30 +184,216 @@ Depending on the distribution you use, it might be possible to install lldap
 from a package repository, officially supported by the distribution or
 community contributed.
 
-#### Debian, CentOS Fedora, OpenSUSE, Ubuntu
-
-The package for these distributions can be found at [LLDAP OBS](https://software.opensuse.org//download.html?project=home%3AMasgalor%3ALLDAP&package=lldap).
-- When using the distributed package, the default login is `admin/password`. You can change that from the web UI after starting the service.
```

Comment:
I think this got lost in the refactoring

Reply:
You are right, line 190 was lost. (the OBS repository is mentioned in each distro)
Do you want me to change the wording?
There was discussion about it here https://github.com/lldap/lldap/discussions/765#discussioncomment-7918023

Reply:
Yeah, that's a possibility... I'll have to think about the UX. In the meantime, let's keep mentioning the admin/password.